client/src/Components/RingdownCard.js

```diff
@@ -38,9 +39,23 @@ function RingdownCard({ children, className, ringdown, dismissable, onStatusChan
     currentDeliveryStatus === Status.OFFLOADED ? (
       <RingdownBadge status={currentDeliveryStatus} />
     ) : (
-      <Timestamp className="ringdown-card__status" label="ETA:" time={etaDateTimeLocalObj} />
+      <Timestamp
+        className="ringdown-card__status"
+        label={Status.ARRIVED ? 'Arrived At:' : 'ETA:'}
+        time={Status.ARRIVED ? DateTime.fromISO(timestamps.ARRIVED) : etaDateTimeLocalObj}
+      />
     );
 
+  // Below is a secondary option that may provide more clarity than the solution above if we reassign drawerTitle to be a let variable.
+  // switch(currentDeliveryStatus) {
+  //   case Status.OFFLOADED:
+  //     drawerTitle =  <RingdownBadge status={currentDeliveryStatus} />;
+  //   case Status.ARRIVED:
+  //     drawerTitle =  <Timestamp className="ringdown-card__status" label="Arrived At:" time={timestamps.ARRIVED} />
+  //   default:
+  //     drawerTitle =  <Timestamp className="ringdown-card__status" label="ETA:" time={etaDateTimeLocalObj} />
+  // }
+
```

Comment:
I don't mind the uncommented version.  Either way something is repeated, either the `Status.ARRIVED` check or the creation of a `<Timestamp>` component, and the first block is a little more compact.